src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -284,6 +309,7 @@ public void getProjectByUuidTest() {
                 .withMatcher("childUuid", equalTo(childProject.getUuid().toString()))
                 .isEqualTo("""
                         {
+                          "accessTeams": [],
```

Comment:
I'd rather not include this field in responses. You can avoid having `accessTeams` returned in responses by annotating `Project.getAccessTeams` with `@JsonIgnore`, and `Project.setAccessTeams` with `@JsonSetter`.

Reply:
I have found a way to revert these changes but still be able to return accessTeams, but only if its not empty. I think for the future that would be helpful, for some things like seeing inside a project, which teams its corresponds to, so you also can change it easily. This would also allow to sort/search for teams.

Reply:
I think in general it would be best to simply not return the teams here *at all*. [`Team` records may include `apiKeys`](https://github.com/stevespringett/Alpine/blob/953ad483b1b12195250732724475aea96fa23ff1/alpine-model/src/main/java/alpine/model/Team.java#L94-L96), and those should not be leaked, given this endpoint is accessible with lower privileges than the `/api/team` endpoints. 

Also, this endpoint already tends to return way too much data, so I'm being extra cautious with adding more fields to it, in particular if those fields are a potentially unbounded collection.

We can always spend time on making `accessTeams` safe to return here once we have a definitive use-case for it. For the time being, please ensure it's not returned.

Reply:
>  [Team records may include apiKeys](https://github.com/stevespringett/Alpine/blob/953ad483b1b12195250732724475aea96fa23ff1/alpine-model/src/main/java/alpine/model/Team.java#L94-L96)

@nscuro I just realized that you get the API keys also over the new /team/visible Endpoint, which is open to all authenticated users.